theme/theme.scss

```diff
@@ -29,3 +29,14 @@
 [class*="ck-heading_heading"] {
 	font-weight: bold;
 }
+
+// Resize dropdown's button label.
+.ck-dropdown {
+	&.ck-heading-dropdown {
+		.ck-dropdown__button {
+			.ck-button__label {
+				width: 8em;
```

Comment:
OK, I recall now the issue. I forgot that the label will change when the selection changes. So if possible labels exceed that min width, it would cause constant changes in the buttons' placement in the toolbar. That would be really disturbing.